### DISCUSSION TITLE
Recommended way to join lines

### DISCUSSION THREAD

**Discussion by jgarte** [meow-edit/meow#456] [Q&A] [answered]
Hi,

What is the recommended way to join the two dangling parentheses at the end of the code snippet with the above sexp form in one or two keystrokes?

In vis I would just do `gJ` if my cursor is on the line with the `setq` form.

Before:

```lisp
           (setq auth-source-pass-filename "~/.password-store")
           ))
```

After:

```lisp
           (setq auth-source-pass-filename "~/.password-store")))
```

One way I can imagine doing this is by typing `xdddddddddd` on the setq form line or `jms`?

**Comment by DogLooksGood** [comment, marked as answer]
`meow-join`, `meow-kill`. 

PS:
1. You can use `negative-argument` before `meow-join` for reversed selection.
2. `meow-join` is expandable. Expand to select all continuous empty lines.

**Reply by jgarte** [reply]
Thanks! Cool, yeah. I had figured it out. 

`jms` was my `meow-next`, `meow-join, and finally `meow-kill` since I am using the qwerty layout mostly.

![output-2023-05-07-22:47:04](https://user-images.githubusercontent.com/47760695/236730004-515a2d1b-20bb-4988-b562-5e5a62c12c0b.gif)
